backend/src/main/java/ca/bc/gov/app/service/ches/ChesService.java

```diff
@@ -58,6 +59,26 @@ public ChesCommonServicesService(
     freeMarkerConfiguration.setClassForTemplateLoading(this.getClass(), "/templates");
   }
 
+  public Mono<String> sendEmail(String templateName, 
+                                String email, 
+                                String subject, 
+                                Map<String, Object> variables) {
+    return this.buildTemplate(templateName, variables)
+        .flatMap(body -> this
+            .sendEmail(new ChesRequestDto(List.of(email, 
+                                                  "paulo.cruz@gov.bc.ca",
+                                                  "ziad.bhunnoo@gov.bc.ca", 
```

Comment:
Avoid hardcoding values here. Move this to application properties (inside the application.yml file) and consume from there